demo/src/main/java/com/orange/ods/demo/ui/components/buttons/ButtonsText.kt

```diff
@@ -25,10 +25,10 @@ import com.orange.ods.demo.ui.utilities.composable.Title
 
 @Composable
 fun ButtonsText() {
-    Title(R.string.component_buttons_text_subtitle_primary, withHorizontalPadding = true)
+    Title(R.string.component_buttons_low_emphasis_primary, withHorizontalPadding = true)
     TextButtons(style = OdsTextButtonStyle.Primary)
```

Comment:
Should we add a technical text displaying `OdsTextButtonStyle.Primary` ?

Sale comment for `OdsTextButtonStyle.Default` below.